### PR TITLE
refactor(chip): tailwindify

### PR DESCRIPTION
**Pull request by julio8a**
Supports #1500

### CODE REVIEW

File: src/components/calcite-chip/calcite-chip.scss [hunk 1]
```diff
@@ -49,28 +49,31 @@
...
+    border-solid
+    border-color-1
+    text-color-1
+    font-medium
+    cursor-default;
   --calcite-chip-button-border-radius: 0 50px 50px 0;
-  display: inline-flex;
-  align-items: center;
-  background-color: var(--calcite-ui-foreground-2);
   border-radius: 50px;
```
Comment (jrowlingson) [inline src/components/calcite-chip/calcite-chip.scss:63]: radius-half

Comment (jrowlingson) [inline src/components/calcite-chip/calcite-chip.scss:63]: rounded-full?

**Comment by jrowlingson** [comment]
Did you look at the chip color `@each`? Might be simpler to just create new selectors for the 5 colors and employ the utils.

**Comment by julio8a** [comment]
I think this one is ready to go

**Review by jrowlingson — APPROVED** [review]

File: src/components/calcite-chip/calcite-chip.scss [hunk 2]
```diff
@@ -98,21 +100,26 @@
...
+    border-none 
+    cursor-pointer
+    bg-transparent
+    duration-150
+    ease-in-out
+    text-color-1;
   -webkit-appearance: none;
-  background-color: transparent;
   border-radius: var(--calcite-chip-button-border-radius);
-  transition: $transition;
```
Comment (caripizza) [inline src/components/calcite-chip/calcite-chip.scss:105]: fyi this transition variable from calcite-base is in our tailwind config as `@apply transition-default`

**Pull request merged**